### PR TITLE
Spelling and grammar corrections

### PR DESCRIPTION
**Pull request by Shubham-1999**

### CODE REVIEW

**Comment by finos-admin** [comment]
Thank you for your contribution and **Welcome to our Open Source Community!**

To make sure your pull request is accepted successfully, we ask all our open source contributors to sign a [Contributor License Agreement](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Legal+Requirements#LegalRequirements-ContributorLicenseAgreement).

Having reviewed our contributor list, we require a CLA for the following people : (@Shubham-1999).

If you need help obtaining a CLA, please read the [Requirements for Contributions](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Legal+Requirements#LegalRequirements-ContributorLicenseAgreement) section of our CLA wiki or email [help@finos.org](mailto:help@finos.org) with your questions.

Thanks once again for your contribution. Let us work with you to make the CLA process quick, easy and efficient so we can move forward with reviewing and accepting your pull request.

_cc_ **@finos-admin**

File: docs/md/development.md [hunk 1]
```diff
@@ -211,10 +211,10 @@ installed in a Docker container.
 The Puppeteer/UI tests are a form of
 [characterization tests](https://en.wikipedia.org/wiki/Characterization_test)
 which use screenshots to compare current and previous behavior of
-`<perspective-viewer>` and its plugins. The results of the each comparison are
+`<perspective-viewer>` and its plugins. The results of  each comparison are
```
Comment (timkpaine) [inline docs/md/development.md:214]: 2 spaces here

**Comment by finos-admin** [comment]
Thank you for your contribution and **Welcome to our Open Source Community!**

To make sure your pull request is accepted successfully, we ask all our open source contributors to sign a [Contributor License Agreement](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Legal+Requirements#LegalRequirements-ContributorLicenseAgreement).

Having reviewed our contributor list, we require a CLA for the following people : (@Shubham-1999).

If you need help obtaining a CLA, please read the [Requirements for Contributions](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Legal+Requirements#LegalRequirements-ContributorLicenseAgreement) section of our CLA wiki or email [help@finos.org](mailto:help@finos.org) with your questions.

Thanks once again for your contribution. Let us work with you to make the CLA process quick, easy and efficient so we can move forward with reviewing and accepting your pull request.

_cc_ **@finos-admin**

File: docs/md/development.md (hunk 1 above)
Comment (Shubham-1999) [reply docs/md/development.md]: Removed spaces.

**Comment by texodus** [comment]
Thanks for the PR!

**Pull request merged**